src/euphorie/client/model.py

```diff
@@ -799,6 +799,20 @@ def refresh_survey(self, survey=None):
                     item.title = zodb_item.title
         self.refreshed = datetime.datetime.now()
 
+    def update_measure_types(self, survey):
```

Comment:
```suggestion
    def update_measure_types(self, survey):
        """ Update measure type to ``in_place_custom`` if the type was
         ``in_place_standard`` but the measure deleted.
         This allows for showing deleted measures in old sessions where they were used.
         """
```

Reply:
More context information on what this method does. Not sure if the wording is correct.
The method name is a bit too generic for the specific purpose it serves, but feel free to keep it as it is.